crates/top/rerun/src/commands/entrypoint.rs

```diff
@@ -630,6 +631,44 @@ where
             }
         }
     } else {
+        #[cfg(all(not(target_arch = "wasm32"), feature = "perf_telemetry"))]
+        let mut _telemetry = {
+            // Safety: anything touching the env is unsafe, tis what it is.
+            #[expect(unsafe_code)]
+            unsafe {
+                std::env::set_var("OTEL_SERVICE_NAME", "rerun");
+            }
+
+            // NOTE: We're just parsing the environment, hence the `vec![]` for CLI flags.
+            use re_perf_telemetry::external::clap::Parser as _;
+            let args = re_perf_telemetry::TelemetryArgs::parse_from::<_, String>(vec![]);
```

Comment:
when _are_ we actually using all the telemetry parameters? in redap?

Reply:
yes